UPGRADE-2.3.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 2.3.3 to 2.3.4
```

Comment:
Sorry I no date to give.

But be sure I want to look at the improvement of release management with @rande soon.